src/wallet/wallet.cpp

```diff
@@ -1418,6 +1429,13 @@ bool CWallet::IsHDEnabled() const
     return !hdChain.seed_id.IsNull();
 }
 
+bool CWallet::IsBlank()
+{
+    // A Blank wallet either has the flag set or it has a version number requiring HD but no HD seed is set
+    LOCK(cs_wallet);
+    return IsWalletFlagSet(WALLET_FLAG_BLANK_WALLET) || (CanSupportFeature(FEATURE_HD) && !IsHDEnabled());
```

Comment:
In commit "[wallet] Support creating a blank wallet" (1cdb8d8346649434450e410971cc3d68c3504a25)

If there a wallet was created after the FEATURE_HD version but with `-usehd=0` set, would it cause this function to return true and no longer let GetKeyFromPool and TopUpKeyPool work?

Reply:
I don't think so because those wallets would have a version less than FEATURE_HD.